app/assets/javascripts/application.js

```diff
@@ -12,5 +12,3 @@
 //= require transactions
 //= require support
 //= require_tree ./modules
-
-//= require ./views/save-location
```

Comment:
Why has this been removed? I've just checked the deployment and it looks like the script no longer runs. Is it included somewhere else somehow or is this a mistake?

Reply:
It's now moved to modules (see: https://github.com/alphagov/frontend/pull/2936/commits/de878c157cde92137ca4648f90fad8fb640dda21) so automatically required by line 14.

It might not have worked for you because the include for this is inside an AB test. As far as I can tell it's all working correctly.

Reply:
Ah of course. Sorry.